common-adapter-moshi/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply from: '../buildsystem/coverageReport.gradle'
+
+sourceCompatibility = JavaVersion.VERSION_1_8
+targetCompatibility = JavaVersion.VERSION_1_8
+
+dependencies {
+    // We specify junit before anything else to make sure that this version is prioritized over the
+    // one bundled in the android.jar
+    testImplementation "junit:junit:$versions.junit"
+
+    implementation project(":models")
+    implementation project(":utils")
+    implementation project(":crypto")
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib"
```

Comment:
This should probably look like `implementation "org.jetbrains.kotlin:kotlin-stdlib:$versions.kotlin"`